lib/platform-stats.js

```diff
@@ -1,5 +1,6 @@
 import assert from 'node:assert'
 import createDebug from 'debug'
+import pg from 'pg'
```

Comment:
This is not used anywhere at runtime, is it? You should import `pg` inside the type declaration only.

Reply:
Gotcha. I have since moved this to just index.js by the way, I think this is the older commit